src/consts/mpich.jl

```diff
@@ -224,7 +224,7 @@ const MPI_Win_errhandler_fn = MPI_Win_errhandler_function
 @const_ref MPI_ARGV_NULL      Ptr{Cvoid} C_NULL
 @const_ref MPI_ARGVS_NULL     Ptr{Cvoid} C_NULL
 
-@const_ref MPI_UNWEIGHTED     Ptr{Cvoid} cglobal((:MPI_UNWEIGHTED, libmpi), Ptr{Cvoid})
+@const_ref MPI_UNWEIGHTED     Ptr{Cvoid} unsafe_load(cglobal((:MPI_UNWEIGHTED, libmpi), Ptr{Cvoid}))
 @const_ref MPI_WEIGHTS_EMPTY  Ptr{Cvoid} cglobal((:MPI_WEIGHTS_EMPTY, libmpi), Ptr{Cvoid})
```

Comment:
```suggestion
@const_ref MPI_WEIGHTS_EMPTY  Ptr{Cvoid} unsafe_load(cglobal((:MPI_WEIGHTS_EMPTY, libmpi), Ptr{Cvoid}))
```
It looks like this should be changed as well:
https://github.com/pmodels/mpich/blob/main/src/include/mpi.h.in#L388-L389